File: mybatis-generator-core/src/main/java/com/wvkity/mybatis/code/make/utils/SystemUtil.java
package com.wvkity.mybatis.code.make.utils;

/**
 * 系统工具
 * @author wvkity
 */
public final class SystemUtil {

    private SystemUtil() {
    }

    /**
     * 用户主目录
     */
    public static final String USER_HOME = System.getProperty("user.home");

    /**
     * 当前程序所在目录
     */
    public static final String USER_DIR = System.getProperty("user.dir");
}

File: mybatis-generator-gui/src/main/kotlin/com/wkit/lost/mybatis/generator/utils/SystemUtil.kt
package com.wkit.lost.mybatis.generator.utils;

/**
 * 系统工具
 * @author wvkity
 */
class SystemUtil {

    companion object {
        /**
         * 获取用户主目录
         * @return 目录地址
         */
        fun userHome(): String {
            return System.getProperties().getProperty("user.home")
        }

        fun userDir(): String {
            return System.getProperties().getProperty("user.dir")
        }
    }
}
